src/components/config_profile/test/profile_test.cc

```diff
@@ -724,6 +724,22 @@ TEST_F(ProfileTest, CheckIntContainerInSecurityData) {
 }
 #endif
 
+TEST_F(ProfileTest, AppIconsFolder_ValuesFromIni_SUCCESS) {
+  profile_.set_config_file_name("smartDeviceLink_test.ini");
+  EXPECT_EQ("smartDeviceLink_test.ini", profile_.config_file_name());
+  EXPECT_EQ(file_system::CurrentWorkingDirectory() + "/storage",
+            profile_.app_icons_folder());
+  EXPECT_EQ(104857611u, profile_.app_icons_folder_max_size());
+}
+
+TEST_F(ProfileTest, AppIconsFolder_DefaultValues_SUCCESS) {
+  profile_.set_config_file_name("smartDeviceLink.ini");
+  EXPECT_EQ("smartDeviceLink.ini", profile_.config_file_name());
+  EXPECT_EQ(file_system::CurrentWorkingDirectory(),
+            profile_.app_icons_folder());
+  EXPECT_EQ(104857600u, profile_.app_icons_folder_max_size());
```

Comment:
@okozlovlux This value is mismatching with the value in smartDeviceLink.ini file (AppIconsFolderMaxSize = 104857500), shouldn't it be the same?